proto/anki/links.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+syntax = "proto3";
+
+package anki.links;
+
+import "anki/generic.proto";
+
+service LinksService {
+  rpc HelpPageLink(HelpPageLinkRequest) returns (generic.String);
+}
+
+message HelpPageLinkRequest {
+  enum HelpPage {
+    NOTE_TYPE = 0;
+    BROWSING = 1;
+    BROWSING_FIND_AND_REPLACE = 2;
+    BROWSING_NOTES_MENU = 3;
+    KEYBOARD_SHORTCUTS = 4;
+    EDITING = 5;
+    ADDING_CARD_AND_NOTE = 6;
+    ADDING_A_NOTE_TYPE = 7;
+    LATEX = 8;
+    PREFERENCES = 9;
+    INDEX = 10;
+    TEMPLATES = 11;
+    FILTERED_DECK = 12;
+    IMPORTING = 13;
+    CUSTOMIZING_FIELDS = 14;
+    DECK_OPTIONS = 15;
+    EDITING_FEATURES = 16;
+  }
+  oneof page {
+    HelpPage variant = 1;
+    string literal = 2;
+  }
```

Comment:
This is a totally valid approach, and fine as is, but I'd be tempted to drop the string case, allowing us to use the enum directly instead of having to deal with a oneof. The use of string arguments in the Python end is us supporting legacy code rather than something we actually want to encourage, and we could keep handling that legacy path in the Python end. Not a big deal though.

Reply:
No, I agree. If we don't do anything special with the literal, it's a lot cleaner to drop the oneof and handle it on the frontend.